MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+       String lowered = "";
+        for (int i = 0; i < str.length(); i++) {
+            if(str.charAt(i) >= 97 && str.charAt(i) <= 122){
+                lowered += str.charAt(i);
+            } else if(str.charAt(i) >= 65 && str.charAt(i) <= 90){
+                lowered +=(char) ((int) str.charAt(i)+ 32);
+            } else if (str.charAt(i) == ' ' ||(str.charAt(i) >= 48 && str.charAt(i) <= 57)){
+                lowered += str.charAt(i);
```

Comment:
1. I would rather you use ASCII chars rather than numbers. 
2. Sometimes `+ 32` isn't clear enough.. use `- 'A' + 'a' ` to make things more clear for to lowercase formula.